proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties

```
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Du bist bereits mit diesem Server verbunden!
+velocity.error.already-connected-proxy=Du bist bereits mit diesem Proxy verbunden!
+velocity.error.already-connecting=Du versuchst bereits eine Verbindung mit dem Server herzustellen!
+velocity.error.cant-connect=Keine Verbindung möglich zu {0}: {1}
+velocity.error.connecting-server-error=Es ist derzeit nicht möglich eine Verbindung mit {0} herzustellen. Bitte versuche es später erneut.
+velocity.error.connected-server-error=Bei der Verbindung zu {0} ist ein Problem aufgetreten.
+velocity.error.internal-server-connection-error=Bei der Verbindung mit dem Server ist ein interner Fehler aufgetreten.
+velocity.error.logging-in-too-fast=Du meldest dich zu schnell an, versuche es später noch einmal.
+velocity.error.online-mode-only=Du bist nicht in deinem Minecraft Konto eingeloggt. Wenn du in deinem Minecraft Konto eingeloggt bist, versuche dein Minecraft Client neu zu starten.
+velocity.error.player-connection-error=Bei deiner Verbindung ist ein interner Fehler aufgetreten.
+velocity.error.modern-forwarding-needs-new-client=Dieser Server ist nur mit Minecraft 1.13 und höher kompatibel.
+velocity.error.modern-forwarding-failed=Dein Server hat keine Weiterleitungsanforderung an den Proxy gesendet. Stelle sicher, dass der Server für die Velocity Weiterleitung konfiguriert ist.
+velocity.error.moved-to-new-server=Du wurdest von {0} vom Server geworfen: {1}
+velocity.error.no-available-servers=Es gibt keine verfügbaren Server mit denen du dich verbinden kannst. Versuche es später erneut oder kontaktiere einen Admin.
+
+# Commands
+velocity.command.generic-error=Beim Ausführen des Befehls ist ein Fehler aufgetreten.
+velocity.command.command-does-not-exist=Dieser Befehl existiert nicht.
+
+velocity.command.players-only=Nur Spieler können diesen Befehl ausführen.
+velocity.command.server-does-not-exist=Der angegebene Server {0} existiert nicht.
+
+velocity.command.server-current-server=Du bist derzeit verbunden mit {0}.
+velocity.command.server-too-many=Es sind zu viele Server eingerichtet. Verwenden Sie die Tabulatorvervollständigung, um alle verfügbaren Server aufzulisten.
```

This is the only place you used `Sie` and I've never seen anyone actually use the full word "Tabulator" 👀.
```suggestion
velocity.command.server-too-many=Es sind zu viele Server eingerichtet. Verwende die Tabvervollständigung, um alle verfügbaren Server aufzulisten.
```